model/riscv_insts_vext_fp.sail

```diff
@@ -0,0 +1,1311 @@
+/* ******************************************************************************* */
+/* This file implements part of the vector extension.                              */
+/* Chapter 13: Vector Floating-Point Instructions                                  */
+/* Chapter 16: Vector Permutation Instructions (floating-point part)               */
+/* ******************************************************************************* */
```

Comment:
#145 Is not addressed here, I wonder if its something that we need to be checking for when accepting new PRs @billmcspadden-riscv  !

Reply:
Copyright headers have been added for vector extension code.